flume/neore/scripts/package.sh

```diff
@@ -98,14 +99,7 @@ function copy_cygnus_conf() {
 	_logStage "######## Copying cygnus template config files to destination config directory... ########"
 	rm -rf {RPM_SOURCE_DIR}/config 
 	mkdir -p ${RPM_SOURCE_DIR}/config
-	for file in $(ls ${BASE_DIR}/conf)
-	do
-		# file=$(basename ${file})
-		# renamed_file=${file%.template}
-		# cp ${BASE_DIR}/conf/${file} ${RPM_SOURCE_DIR}/config/${renamed_file}
-
-		cp ${BASE_DIR}/conf/${file} ${RPM_SOURCE_DIR}/config/
-	done
+        cp ${BASE_DIR}/conf/* ${RPM_SOURCE_DIR}/config/
```

Comment:
What was done here was to remove unnecesary loop and leftover comments.

Regarding the indentantion issue, uff, my IDE shows everything OK... The problem is the orginal file was indented by using a single "big" tab that is equivalent to 2 tabs in my IDE (Netbeans).